src/dagmc/test/dagmc_unit_test_driver.cc

```diff
@@ -3,7 +3,18 @@
 
 #include <gtest/gtest.h>
 
+
+int num_thread_to_run; // number of threads for the test
+
 int main(int argc, char* argv[]) {
+
+  // set the number of threads
+  if ( argc > 1 ) {
+    num_thread_to_run = std::atoi(argv[1]);
```

Comment:
This is that line causing some of the unit tests to fail.

Reply:
Ok, that one can be fixed.